app/Services/Email.php

```diff
@@ -202,13 +203,18 @@ protected function getLeaderboardVars($users)
      */
     protected function getFeaturedReportback()
     {
-        if (isset($this->message->reportback_id) && isset($this->message->reportback_item_id)) {
-            $featuredReportback = $this->manager->appendReportbackItemToMessage($this->message->reportback_id, $this->message->reportback_item_id);
+        if (! isset($this->competition)) {
+            return;
+        }
+
+        $featuredReportback = FeaturedReportback::where('competition_id', '=', $this->competition->id)->where('message_id', '=', $this->message->id)->first();
+        if (isset($featuredReportback->reportback_id) && isset($featuredReportback->reportback_item_id)) {
```

Comment:
Have you tried hitting this part if `$featuredReportback` is actually `null` (didn't find it in the DB?). Just want to make sure the `isset()` logic here works if it's checking for a property on `null`.